### ISSUE TITLE
1.1.0 crashing on plan

### ISSUE THREAD

**Issue by drdamour** [microsoft/terraform-provider-azuredevops#1047]
Not sure where it's crashing our IaC is a few thousand lines at this point.  i'll keep looking and update as i figure it out.

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Affected Resource(s)

* `azuredevops_XXXXX`

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```

### Panic Output

```
Stack trace from the terraform-provider-azuredevops_v1.1.0 plugin:

panic: interface conversion: interface {} is nil, not string

goroutine 17876 [running]:
github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/service/core.readSubjectDescriptors(0xc0001d1e00, 0xc0007be048)
        github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/service/core/resource_team.go:611 +0x2fd
github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/service/core.readTeamMembers(0xc0001d1e00, 0xc000523300)
        github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/service/core/resource_team.go:364 +0xf0
github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/service/core.resourceTeamRead(0xc000633d00, {0x15d2720?, 0xc0001d1e00})
        github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/service/core/resource_team.go:167 +0x27d
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0x1922b40?, {0x1922b40?, 0xc000a6aed0?}, 0xd?, {0x15d2720?, 0xc0001d1e00?})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.23.0/helper/schema/resource.go:712 +0x178
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc000467ce0, {0x1922b40, 0xc000a6aed0}, 0xc00073d1e0, {0x15d2720, 0xc0001d1e00})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.23.0/helper/schema/resource.go:1015 +0x585
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0xc000005530, {0x1922b40?, 0xc000a6adb0?}, 0xc000a840c0)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.23.0/helper/schema/grpc_provider.go:613 +0x497
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ReadResource(0xc0003c6be0, {0x1922b40?, 0xc000a6a7e0?}, 0xc000f05c80)
        github.com/hashicorp/terraform-plugin-go@v0.14.0/tfprotov5/tf5server/server.go:748 +0x49e
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadResource_Handler({0x1759160?, 0xc0003c6be0}, {0x1922b40, 0xc000a6a7e0}, 0xc00033d9d0, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.14.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:349 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000001e0, {0x19258e0, 0xc0005841a0}, 0xc000911560, 0xc00049a870, 0x1eca530, 0x0)
        google.golang.org/grpc@v1.56.3/server.go:1335 +0xde3
google.golang.org/grpc.(*Server).handleStream(0xc0000001e0, {0x19258e0, 0xc0005841a0}, 0xc000911560, 0x0)
        google.golang.org/grpc@v1.56.3/server.go:1712 +0xa1b
google.golang.org/grpc.(*Server).serveStreams.func1.1()
        google.golang.org/grpc@v1.56.3/server.go:947 +0xca
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.56.3/server.go:958 +0x15c

Error: The terraform-provider-azuredevops_v1.1.0 plugin crashed!
```

### Steps to Reproduce

1. `terraform apply`

### References

* #0000

**Comment by drdamour** [comment]
@xuzhang3 r we getting a 1.1.1 release with this fix?  1.1.0 is unusable for me.

**Comment by drdamour** [comment]
oh...hm somethign weird going on with https://github.com/microsoft/terraform-provider-azuredevops/releases/tag/v1.1.1

notice the title is 1.1.0 but the tag is 1.1.1

<img width="713" alt="Screen Shot 2024-05-16 at 7 41 42 AM" src="https://github.com/microsoft/terraform-provider-azuredevops/assets/1514496/8d711cb2-a496-407e-8a19-2b823c3e6a33">

**Comment by xuzhang3** [comment]
@drdamour  are you able to pull the latest release from Terraform registry?  The release title has been corrected to v1.1.1

**Comment by drdamour** [comment]
@xuzhang3 yup just got to try it out and working now, thanks for the quick turnaround on this one!
